content/en/platform/corda/4.9/enterprise/cordapps/debugging-a-cordapp.md

```diff
@@ -38,7 +38,7 @@ running via the node driver to debug your CorDapp. You can debug the CorDapp:
 
 ### Debug with the nodes in-process
 
-1. [Define a network using the node driver](../../../../../../en/tutorials/corda/4.9/os/supplementary-tutorials/tutorial-integration-testing.md).
+1. [Define a network using the node driver](../../../../../tutorials/corda/4.9/community/supplementary-tutorials/tutorial-integration-testing.md).
```

Comment:
deliberately community, I presume?

Reply:
Yup, the original hadn't been updated from the old "os" format.